pkg/views/profile/create.go

```diff
@@ -29,8 +29,9 @@ func ProfileCreationView(c *config.Config, profileAddView *ProfileAddView, editi
 			if str == "" {
 				return errors.New("profile name can not be blank")
 			}
-			if match, _ := regexp.MatchString("^[a-zA-Z0-9]+$", str); !match {
-				return errors.New("profile name must be alphanumeric only")
+			result, err := util.GetValidatedName(str)
```

Comment:
The usage of this function is not ideal here since it modifies the `str` which then would have to be set on the input.

The same function can not be reused but I get what you were aiming for.